winml/lib/Api.Image/ImageConverter.cpp

```diff
@@ -7,15 +7,10 @@
 #include "inc/D3DDeviceCache.h"
 
 using namespace Microsoft::WRL;
-using namespace Windows::Graphics::DirectX::Direct3D11;
-using namespace Windows::AI::MachineLearning::Internal;
-using namespace winrt::Windows::AI::MachineLearning::implementation;
-using namespace winrt::Windows::Media;
-using namespace winrt::Windows::Graphics::Imaging;
-using namespace winrt::Windows::Graphics::DirectX;
-using namespace winrt::Windows::Graphics::DirectX::Direct3D11;
-
-void ImageConverter::SyncD3D11ToD3D12(_In_ D3DDeviceCache& device_cache, _In_ ID3D11Texture2D* pD3D11Texture) {
+
+using namespace _winml;
```

Comment:
same question as before - do you need the using if you're going to be explicit later?

Reply:
_winml is needed because the class is in the _winml namespace. It is a pain to prefix all of the class member definitions with _winml::, but i want to know where methods called from other parts of the codebase are defined.